src/components/AddExercise.vue

```diff
@@ -441,14 +484,47 @@ export default defineComponent({
       return null;
     },
     enabled(): Exercise[] {
-      return this.exercises.filter(e =>
-        e.trainingDevices.every(t => this.equipments.find(e => e.id == t) && !this.equipments.find(e => e.id == t)?.disabled)
-      );
+      return this.exercises
+        .filter(e => e.trainingDevices.every(t => this.equipments.find(e => e.id == t) && !this.equipments.find(e => e.id == t)?.disabled))
+        .sort(function (a, b) {
+          if (a.name < b.name) {
+            return -1;
+          }
+          if (a.name > b.name) {
+            return 1;
+          }
+          return 0;
+        })
+        .sort(function (a, b) {
+          let map = {
+            easy: 1,
+            medium: 2,
+            hard: 3,
+          };
+          return map[a.difficulty] - map[b.difficulty];
+        });
     },
     disabled(): Exercise[] {
```

Comment:
also this can be combined with the `enabled` computed property, if i see that correctly `disabled `is equivalent to `this.enabled.filter(e => e.trainingDevices.length > 0)`